plugins/woocommerce/includes/class-wc-privacy-exporters.php

```diff
@@ -69,7 +69,7 @@ public static function order_data_exporter( $email_address, $page ) {
 
 		$orders = wc_get_orders( $order_query );
 
-		if ( 0 < count( $orders ) ) {
+		if ( is_array( $orders ) && 0 < count( $orders ) ) {
```

Comment:
This change can be left out. There is no reason that $orders wouldn't be an array unless paginate is passed into wc_get_orders(). Since the code controls those parameters, it shouldn't happen.

Reply:
done https://github.com/woocommerce/woocommerce/pull/41482/commits/9545bec944d821cc723b2528085e71545a2cd172